client/src/components/PacmanGame/PacmanGame.jsx

```diff
@@ -314,6 +321,7 @@ class PacmanGame extends Component {
         pacman: { ...pacman, direction: newDirection },
       });
     }
+    changeDirection({ playerId, direction: newDirection });
```

Comment:
This function **sends new direction** rather than change any direction. Use a better name.